modules/docs/src/main/mdoc/docs/08-Fragments.md

```diff
@@ -76,6 +76,16 @@ count("city").quick.unsafeRunSync()
 ```
 
 > Note that `Fragment.const` performs no escaping of passed strings. Passing user-supplied data is an **injection risk**.
+>
+
+You can also use Fragments inside `fr` interpolator directly. Both parameters and SQL literals will be
```

Comment:
I think it's worth including a "note" section on sql injection, and in what ways this is and is not safe, injection-wise

Reply:
I think this it is only a concern for Fragment.const right? Perhaps we need more emphesis there?

Reply:
I think just to mention that as long as the interpolated fragment is constructed safely, then this is safe.

Reply:
Sure I'll see how I can fit it in. (Though I'm not sure how it is different from `++`)

Reply:
done